lib/tether/ops_linux.go

```diff
@@ -699,6 +700,52 @@ func (t *BaseOperations) MountTarget(ctx context.Context, source url.URL, target
 	return nil
 }
 
+// CopyExistingContent copies the underlying files shadowed by a mount on a directory
+// to the volume mounted on the directory
+// see bug https://github.com/vmware/vic/issues/3482
+func (t *BaseOperations) CopyExistingContent(source string) error {
+	defer trace.End(trace.Begin(fmt.Sprintf("copyExistingContent from %s", source)))
+
+	source = filepath.Clean(source)
+
```

Comment:
We would need to bind anyways mount to check if there are any files in the underlying directory 

To avoid doing a bind mount, we could change the code flow to the steps below 
1. In mountLabel and mountTarget, check if the original folder is empty 
2. Mount the volume 
3. If step 1 is false then go ahead with copy